verso/Cargo.toml

```diff
@@ -8,3 +8,6 @@ ipc-channel = { workspace = true }
 serde = { workspace = true }
 url = { workspace = true }
 versoview_messages = { path = "../versoview_messages" }
+
+[features]
+pre-built-versoview = []
```

Comment:
I don't think this needs to be a feature flag. Just an env variable should be enough.

Reply:
I'm thinking that when this feature is on, we can provide something in verso that exposes this path, so the consumer can easily turn this feature on and don't have to worry about the versoview

Reply:
Well, after thinking about it a bit more, this doesn't quite make sense since the path is not stable when they distribute the app, environment variables should work better here